src/utils/dig.js

```diff
@@ -92,6 +98,7 @@ export const parseEmbed = embed => {
         name: descMatch[1],
         type: descMatch[2],
         short: !!descMatch[4],
+        cdflag: descMatch[5],
```

Comment:
```suggestion
        cdflag: !!descMatch[5],
```
Typecast to a boolean here